app/controllers/downloads_controller.rb

```diff
@@ -158,7 +156,11 @@ def create_mobi_html
   def render_mobi_html(template, basename)
     @mobi = true
     html = render_to_string(:template => "downloads/#{template}.html", :layout => 'barebones.html')
-    html = Iconv.conv("ASCII//TRANSLIT//IGNORE", "UTF8", html)
+    html.encode!(Encoding::UTF_8,
+                Encoding::ISO_8859_1,
+                :invalid => :replace,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.